_posts/2022-07-01-So-You-Want-To-eWaterCycle.md

```diff
@@ -0,0 +1,75 @@
+
+![alt_text](/assets/So_You_Want_To_eWaterCycle_diagram.png "Flowchart")
+
+### A quick look at eWaterCycle on our demonstration machine
+
+If you want to get a feel for the eWaterCycle platform we have a small demonstration machine available. You will need a login for this machine which we will email you if you fill in this form. This machine is completely wiped and restarted regularly, including all login information. So don’t get excited and start doing research as this machine is only intended to take a quick look around, generate a hydrograph and play around with the options, not to do research. To get to know the platform we recommend looking at the use cases from the GMD paper which live in their own repository and can be accessed through the “Tech Paper Notebooks” button at the top of the screen in the explorer. Or you can use the explorer to browse the available models and datasets and let the system generate a notebook for you.
+
+### Run eWaterCycle on your own SURF researchCloud account
+
+Researchers with an account[^2] on the [SURF researchCloud](https://portal.live.surfresearchcloud.nl/) can start up their own machine. This machine will out of the box be able to run a few example models and datasets, such as PCRGlobWB and WFLOW for the Rhine Area. Users can add their own models and datasets following the steps explained in the [documentation](https://ewatercycle.readthedocs.io/en/latest/adding_models.html).
+
+Note that multiple users can work on one machine. This is handled by the SURF Research Access Management ([SRAM](https://wiki.surfnet.nl/display/SRAM/)) system.  
+
+### Run eWaterCycle on your own machine
+
+For researchers with access to their own machine there is the option of installing eWaterCycle on it and providing access to users of that machine. This machine should run Linux and have plenty of storage. While in theory this machine could be a laptop, usually this will be a server where users login. Following the steps at [TODO link] provide you with all the needed info.To save many headaches we recommend that Installing eWaterCycle on your own hpc or cloud infra is done by system administrators or research software engineers who are comfortable doing so.
+
+### Models and datasets
+
+All of the above ways of working with eWaterCycle will give you access to the models and datasets that ‘ship’ with the package. On our demonstration machine we have added a few more datasets, see [https://www.ewatercycle.org/demo](https://www.ewatercycle.org/demo) for details. eWaterCycle is a work in progress and we keep adding models and datasets as we move forward. 
```

Comment:
```suggestion
We aim to provide example datasets (parameters, forcing) that can be automatically downloaded for all supported models in eWaterCycle. More datasets can be added in the process of setting up the platform. On our demonstration machine we have added several additional datasets, see [https://www.ewatercycle.org/demo](https://www.ewatercycle.org/demo) for details. eWaterCycle is a work in progress and we keep adding models and datasets as we move forward. 
```

Reply:
the suggested text focusses on what we aim to do where the original text states what the current package ships with, which is what I wanted the focus to be on. I do like the "More datasets can be added in the process of setting up the platform. " sentence though. Can you change the commit to only say that?

Reply:
yes, that final suggestion is better than both the previous one and my original text, will commit!